.devops/deploy-pipelines.yml

```
@@ -0,0 +1,69 @@
+# Automatically triggered on PR
+# Only triggers on branches
+pr:
+  autoCancel: true
+  branches:
+    exclude:
+     - '*'
+
+trigger:
+  branches:
+    exclude:
+      - master
+  paths:
+    include:
+      - 'src/*'
+      - '.devops/*'
```

```suggestion
    include:
      - 'src/*'
      - '.devops/*'
    exclude:
      - 'src/k8s/*'
```
probably we have to exclude k8s changes that requires
- kubectl
- helm
- run on private self hosted agent